TheEthicalHackerCup/Assets/AttackMinigames/Phishing_Attack/Scripts/CountdownTimer.cs

```diff
@@ -6,7 +6,7 @@
 public class CountdownTimer : MonoBehaviour
 {
     private float currentTime;
-    private float startingTime = 20f;
+    private float startingTime = 35f;
```

Comment:
Review how these time variables in this script are used.
Try to put all variable initializations into the Start() method so that they would be easier to make dynamically settable.

This startingTime var is only used to initialize the currentTime var and never used again. 
Perhaps renaming this var to timeUntilExpiration and then bringing it down to zero in Update() would be a lil more clear

Reply:
Cleared it up so theres not 2 variables for time when not needed